pokemon-collection/src/Home.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import './App.css';
+import { BrowserRouter as Router, Route, Switch, Link } from 'react-router-dom';
+import Pokemon from './Pokemon';
+
+export default class Home extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: {},
+            isLoading: true,
+            hasError: false
+        }
+    }
+
+    componentDidMount() {
+        const pokemonUrl = 'https://pokeapi.co/api/v2/pokemon/?limit=964';
```

Comment:
Heads up, my calls were taking a while, might've had something to do with this 964 limit.